stix2elevator/missing_policy.py

```diff
@@ -1,10 +1,14 @@
+import re
+
 from six import text_type
 
 from stix2elevator.options import get_option_value, info, warn
 
 
 def convert_to_custom_property_name(prop_name):
-    return "x_" + get_option_value("custom_property_prefix") + "_" + prop_name
+    if re.search('[A-F]', prop_name):
```

Comment:
Are there no chances that a property name has a capital letter outside A-F?

```suggestion
    if re.search('[A-Z]', prop_name):
```